### PR TITLE
add test transaction override fields

### PR DESCRIPTION
**Pull request by yding03**
This PR will add test transaction override fields.

### CODE REVIEW

File: examples/transactions.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 var berbix = require("../lib/berbix");
 
-var run = async function() {
+var run = async function () {
```
Comment (benbeltzer) [inline examples/transactions.js:3]: I don't think we need that space there

**Review by benbeltzer — APPROVED** [review]
lgtm, just one comment about the space

File: README.md [hunk 2]
```diff
@@ -106,6 +106,7 @@ Parameters:
 
 - `responsePayload: string` - A string describing the payload type to return when fetching transaction metadata, e.g. "us-dl". See [our testing guide](https://docs.berbix.com/docs/testing) for possible options.
 - `flags: string[]` - An optional list of flags to associate with the transaction (independent of the payload's contents), e.g. ["id_under_18", "id_under_21"]. See [our flags documentation](https://docs.berbix.com/docs/id-flags) for a list of flags.
+- `override_fields: { string => string }` - An optional mapping from a [transaction field](https://docs.berbix.com/reference#gettransactionmetadata) to the desired override value, e.g. `params.override_fields = { "date_of_birth": "2000-12-09" } `
```
Comment (jnnnthnn) [inline README.md:109]:
```suggestion
- `overrideFields: { string => string }` - An optional mapping from a [transaction field](https://docs.berbix.com/reference#gettransactionmetadata) to the desired override value, e.g. `params.overrideFields = { "date_of_birth": "2000-12-09" } `
```

File: lib/berbix.js [hunk 3]
```diff
@@ -251,6 +251,9 @@ Client.prototype.overrideTransaction = function (tokens, params) {
   if (params.flags != null) {
     payload.flags = params.flags;
   }
+  if (params.override_fields != null) {
+    payload.override_fields = params.override_fields;
+  }
```
Comment (jnnnthnn) [inline lib/berbix.js:256]:
```suggestion
  if (params.overrideFields != null) {
    payload.override_fields = params.overrideFields;
  }
```

**Review by jnnnthnn — APPROVED** [review]
Looks good and works as expected! Just one nit below.

**Pull request merged**